### DISCUSSION TITLE
Webhooks Integrity verification

### DISCUSSION THREAD

**Discussion by robotshatemonkeys** [kinngh/shopify-node-express-mongodb-app#59] [Ideas]
Shouldn't  the webhook being verified ? 
https://shopify.dev/docs/apps/webhooks/configuration/https#step-5-verify-the-webhook

on **server index > app.post("/webhooks/:topic"** there is no verification. 
This might cause the app to be rejected

**Comment by kinngh** [comment]
That's taken care of by the Shopify library. When you're using `shopify.webhooks.process`, that's taking care of verification. Besides, all my repos are being used in production and every update is first verified by multiple engineers on live apps to ensure these slips don't happen and are compliant with the current App Store rules. 

Also, I highly recommend you to read through Shopify's docs and the shopify api library docs so these slip ups don't happen^
